resources/qml/TimelineView.qml

```diff
@@ -434,18 +434,32 @@ Page {
 				id: footerContent
 				anchors.left: parent.left
 				anchors.right: parent.right
-
-				Label {
-					id: typingDisplay
-					anchors.left: parent.left
-					anchors.right: parent.right
-					anchors.leftMargin: 10
-					anchors.rightMargin: 10
-
-					color: colors.text
-					text: chat.model ? chat.model.formatTypingUsers(chat.model.typingUsers, colors.window) : ""
-					textFormat: Text.RichText
-				}
+                Rectangle {
+                    id: typingRect
+                    Connections {
+                        target: chat.model
+                        onTypingUsersChanged: {
+                            if (chat.model && chat.model.typingUsers < 1)
+                                typingRect.color = "transparent"
+                            else
+                                typingRect.color = colors.window
+                        }
+                    }
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.leftMargin: 10
+                    anchors.rightMargin: 10
+                    color: "transparent"
```

Comment:
You can use a direct binding to make this condition much simpler and get rid of the explicit connection.

```suggestion
                    anchors.left: parent.left
                    anchors.right: parent.right
                    anchors.leftMargin: 10
                    anchors.rightMargin: 10
                    color: (chat.model && chat.model.typingUsers.length > 0) ? colors.window : "transparent"
```